privacyidea/static/components/config/views/config.resolvers.ldap.html

```diff
@@ -162,6 +162,16 @@
                    placeholder="120"/>
         </div>
     </div>
+    <div class="form-group">
+        <label for="serverpool-persistent" class="col-sm-3 control-label" translate>
+            Per-process server pool
```

Comment:
Put this checkbox under the form-group of retry-rounds and pool skip.

And the sizelimit above the retry-rounds and pool skip.

Also add a ``help-block`` under the checkbox (like in "editable") with the text:

     This activates a persistent server pool between requests.

Reply:
Done in add0f03